### DISCUSSION TITLE
Flux Kustomization Workflow with multiple Kustomization files

### DISCUSSION THREAD

**Discussion by mlschindler** [fluxcd/flux2#3101] [Q&A] [answered]
Hi all.  

I have a use case wherein my team would like to provide a base set of resources from some artifact source (say an S3 bucket) to install a core infrastructure component such as fluentbit etc. What we would like to do is allow teams to provider their own overlays, not necessarily from the same artifact source, but from their own instead. The closest use case coverage I can see is using Git source controller with submodules which solves the problem in a different way. However it would be awesome if I could bootstrap a target k8s cluster(s) with a common set of required components and then teams could then modify them with their own kustomize overlays. Is this possible? Has anyone tried doing something similar? Does this even make sense?

Thanks!

**Comment by stefanprodan** [comment, marked as answer]
Please see this issue https://github.com/fluxcd/source-controller/issues/854#issuecomment-1226152895

**Reply by adawalli** [reply]
Thanks @stefanprodan, that is exactly what we need!

**Reply by mlschindler** [reply]
Thank you @stefanprodan!!!!
